enterprise/server/cmd/ci_runner/main.go

```diff
@@ -1610,7 +1610,8 @@ func writeBazelrc(path, invocationID string) error {
 	if *prNumber == 0 {
 		lines = append(lines, "build --build_metadata=ROLE=CI")
 	} else {
-		lines = append(lines, "build --build_metadata=ROLE=PR")
+		lines = append(lines, "build --build_metadata=ROLE=CI")
```

Comment:
nit: slightly cleaner to move `build --build_metadata=ROLE=CI` back to `lines[0]`, then use `if *prNumber != 0`

Reply:
Done